### PR TITLE
Money fix

### LINKED ISSUE #581: Money Issue
**Describe the bug**
When you play a game in Solo or modes where player per team is 1, on game end you get the coins for team support

**To Reproduce**

**Expected behavior**

**Screenshots**

**System (please complete the following information):**
 - OS: 
 - Server software 
 - Server Version 
 - Plugin Version 

**Additional context**

### PR DESCRIPTION
**Pull request by Xxyuri2005xX**
- Fix Moneys (team-support) give when you play Solo or 1 player-per-team games
- Updated the name of the variables
- Closes #581

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/money/internal/MoneyListeners.java [hunk 1]
```diff
@@ -24,19 +25,36 @@ public void onGameEnd(GameEndEvent e) {
...
-                player.sendMessage ( Language.getMsg ( player, Messages.MONEY_REWARD_WIN ).replace ( "{money}", String.valueOf ( gamewin ) ) );
+            int gameWin = MoneyConfig.money.getInt ( "money-rewards.game-win" );
+            if (gameWin > 0) {
+                BedWars.getEconomy ().giveMoney ( player, gameWin );
+                player.sendMessage ( Language.getMsg ( player, Messages.MONEY_REWARD_WIN ).replace ( "{money}", String.valueOf ( gameWin ) ) );
+            }
+            ITeam bwt = e.getArena().getExTeam(player.getUniqueId());
+            if (bwt != null) {
+                //noinspection deprecation
+                if (bwt.getMembersCache().size() > 1) {
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/money/internal/MoneyListeners.java]: use arena#maxInTeam > 1

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/money/internal/MoneyListeners.java [hunk 2]
```diff
@@ -24,19 +25,36 @@ public void onGameEnd(GameEndEvent e) {
...
             Player player = Bukkit.getPlayer ( p );
             if (player == null) continue;
-            int teammate = MoneyConfig.money.getInt ( "money-rewards.per-teammate" );
-            if (teammate > 0) {
-                BedWars.getEconomy ().giveMoney ( player, teammate );
-                player.sendMessage ( Language.getMsg ( player, Messages.MONEY_REWARD_PER_TEAMMATE ).replace ( "{money}", String.valueOf ( teammate ) ) );
+            ITeam bwt = e.getArena().getExTeam(player.getUniqueId());
+            if (bwt != null) {
+                //noinspection deprecation
+                if (bwt.getMembersCache().size() > 1) {
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/money/internal/MoneyListeners.java]: same here. members cache is dynamic

**Review by andrei1058 — CHANGES_REQUESTED** [review]

**Comment by Xxyuri2005xX** [comment]
updating right now, for tonight i push the update required by you

**Comment by Xxyuri2005xX** [comment]
updated to arena#maxInTeam

**Pull request merged**